Source/Core/VideoCommon/VertexLoader_Color.h

```diff
@@ -4,14 +4,42 @@
 #pragma once
 
 #include "Common/CommonTypes.h"
+#include "Common/EnumMap.h"
+#include "VideoCommon/CPMemory.h"
 #include "VideoCommon/VertexLoader.h"
```

Comment:
I generally leave a blank line between Common headers and VideoCommon headers (particularly when there are multiple of each of them). This isn't super important, but I think it does make things a bit easier to read.